crates/oxide/src/lib.rs

```diff
@@ -468,6 +468,7 @@ pub fn pre_process_input(content: &[u8], extension: &str) -> Vec<u8> {
     use crate::extractor::pre_processors::*;
 
     match extension {
+        "cshtml" | "razor" => Razor.process(content),
```

Comment:
Used both `cshtml` and `razor`. See: https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-9.0